CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contribution guidelines
+
+Here you will find all the necessary information regarding how to create file, write and structure code for this project. Developers are always welcome to contribute to these guidelines with clear examples.
+
+## General guidelines
+- Avoid hard-coding values instead create constant variables
+- Use absolute paths for `import` statements
+- Avoid using negative conditions
+- Avoid using nested ternary, instead use `if`-`else` blocks
+- Leave wide spaces in codes
```

Comment:
??

Reply:
These are general guidelines to write code, discussed with us in the master class

Reply:
Wide spaces as in empty lines to divide the code in sections?

Reply:
Yes